### ISSUE TITLE
Specify that the LDAP Group import search filter field follows RFC 2254

### ISSUE THREAD

**Issue by raphj** [xwikisas/application-ldapuserimport#82]
In the administration section, the format to use for the LDAP Group import search filter field is not documented.
From reading the code, it seems like it follows https://www.rfc-editor.org/rfc/rfc2254

This should be documented.

Can anyone confirm this before we do it?

**Comment by trrenty** [comment]
Indeed, the format should follow the RFC. I updated the translation for the fiield. I'll also update the documentation
